.github/workflows/codegen.yml

```diff
@@ -1,8 +1,6 @@
 name: Codegen
 
 on:
-  push:
-    branches: [ main ]
```

Comment:
I don't want to remove CI from the main branch.  Instead let's figure out how to have it trigger only on the public repo.

Reply:
Reverted change after discussion, now uses `if` checks instead to filter running on commit to main in private repo.